src/index.ts

```diff
@@ -30,19 +30,42 @@ export default class DataSource {
     options['headers']['x-reverse-proxy-ttl'] = options['cacheTime'] / 1000;
     options['headers']['x-mi-cbe'] = CD._hashForRequest(url, options);
 
-    return CD.get(url, options, cb);
+    return await CD.get(url, options);
   }
 
-  getAllRows(params: object, opts = {}) {
+  /**
+   *
+   * @param params
+   * @param opts
+   */
+  async getAllRows(params: object, opts = {}) {
     params['mi_multiple'] = true;
 
     if (opts['headers']) {
       params['mi_include_headers'] = true;
       opts['headers'] = {};
     }
+    const { data } = await this.getRawData(params, opts);
+    return JSON.parse(data);
+  }
+
+  /**
+   *
+   * @param opts
+   */
+  async getMultipleTargets(opts: any = {}) {
+    const params = {
+      mi_multiple: true,
+      mi_include_headers: true,
+    };
+
+    const { method = null } = opts;
+
+    if (!method && method.toLowerCase() !== 'post') {
+      throw new Error('Request method must be POST for getMultipleTargets');
+    }
```

Comment:
this works, just curious why not hard code `method: "post"` as part of the options?

Reply:
I was thinking this, my line of thought was to make it identical to how Solutions currently makes a `CD.getCORS` request when they pass through the whole options object. 

That way if they 'forgot' to do this. I'm fine to make it hardcoded let me know what you all prefer.

Reply:
Makes sense to keep it the same way for consistency.